lib/menu.rb

```diff
@@ -0,0 +1,28 @@
+require 'yaml'
+
+class Menu
+
+  def initialize
+    @chippy_menu
+    @chinese_menu
+  end
+
+  def load_chippy_menu
```

Comment:
Think about the Open/Closed principle.  What happens when I want to add a pizza menu?